### PR TITLE
Feed API Challenge

### PR DESCRIPTION
**Pull request by g5321247**

### CODE REVIEW

File: FeedAPIChallenge/Feed API/FeedIamgeMapper.swift [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+//  FeedIamgeMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by George Liu on 2020/9/13.
+//  Copyright © 2020 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedIamgeMapper {
```
Comment (caiozullo) [inline FeedAPIChallenge/Feed API/FeedIamgeMapper.swift:11]: typo - should be `FeedImageMapper`

**Review by caiozullo — COMMENTED** [review]
Hi @g5321247,

Congratulations, you've passed the challenge 🥇

All tests are passing, so your implementation complies with the specs.

I left just one comment regarding a typo. Feel free to fix and push it again.

Great job! ✅

File: FeedAPIChallenge/Feed API/FeedIamgeMapper.swift (hunk 1 above)
Comment (g5321247) [reply FeedAPIChallenge/Feed API/FeedIamgeMapper.swift]: Hi @caiozullo, I've fixed the typo.

Thanks

**Comment by caiozullo** [comment]
Nice one!

**Pull request closed without merging**